tests/integration_tests/interpreters.py

```diff
@@ -56,13 +56,15 @@ def get_python_interpreter_by_major_version(cls, major):
             "3.7.13",
             "3.7.14",
             "3.7.15",
+            "3.7.17",
```

Comment:
This list keeps growing. Can we cleanup the old ones? Like the two oldest ones for instance? 
Like what are the criteria?

Reply:
Works for me. Can you create a ticket about that, please?